src/main/java/com/khutircraftubackend/seller/SellerDTO.java

```diff
@@ -15,18 +14,8 @@
 @Builder
 public record SellerDTO(
 
-        @NotBlank (message = "Назва компанії не може бути порожнім")
-        @Size (max = 100, message = "Назва компанії не може перевищувати 100 символів")
-        @Pattern(regexp = "^[a-zA-Zа-яА-ЯІіЇїЄєҐґ\\d&,`'\\-\\s]*$", message = "Назва компанії має містити лише " +
-                "українські або англійські літери, цифри і знаки \"&-,'`\"")
-        String companyName,
-
-        @NotBlank (message = "Ім'я не може бути порожнім")
+        @NotBlank (message = "Назва компанії(ПІБ) не може бути порожнім")
         @Size (max = 100, message = "Ім'я не може перевищувати 100 символів")
-        String sellerName,
-
-        @Size (min = 12, max = 13)
-        @Pattern (regexp = "^\\+380\\d{9}$", message = "Номер телефону має бути у форматі +380__________")
-        String phoneNumber
+        String sellerName
```

Comment:
do we want to apply Pattern validation for seller name field?
```java
@Pattern(regexp = "^[a-zA-Zа-яА-ЯІіЇїЄєҐґ\\d&,`'\\-\\s]*$", 
    message = "Назва компанії має містити лише " +
                "українські або англійські літери, цифри і знаки \"&-,'`\"")
```
btw, note the end of the pattern: `*$` , it should be replaced with `+$` 
coz `*$` allows empty string - will be an issue if we remove `@NotBlank` validation